src/scconf/parse.c

```diff
@@ -270,55 +281,52 @@ void scconf_parse_token(scconf_parser * parser, int token_type, const char *toke
 		}
 		item->value.comment = token ? strdup(token) : NULL;
 		break;
-	case TOKEN_TYPE_STRING:
-		{
-			char *stoken = NULL;
+	case TOKEN_TYPE_STRING: {
+		char *stoken = NULL;
 
-			if ((parser->state & (STATE_VALUE | STATE_SET)) ==
-			    (STATE_VALUE | STATE_SET)) {
-				scconf_parse_warning_expect(parser, ";");
-				scconf_parse_reset_state(parser);
-			}
-			if (token && *token == '"') {
-				/* quoted string, remove them */
-				token++;
-				len = strlen(token);
-				if (len < 1 || token[len - 1] != '"') {
-					scconf_parse_warning_expect(parser, "\"");
-				} else {
-					/* stoken */
-					stoken = strdup(token);
-					if (stoken) {
-						stoken[len - 1] = '\0';
-					}
-				}
-			}
-			if (!stoken) {
-				stoken = token ? strdup(token) : NULL;
-			}
-			if (parser->state == 0) {
-				/* key */
-				parser->key = stoken ? strdup(stoken) : NULL;
-				parser->state = STATE_NAME;
-			} else if (parser->state == STATE_NAME) {
-				/* name */
-				parser->state |= STATE_SET;
-				scconf_list_add(&parser->name, stoken);
-			} else if (parser->state == STATE_VALUE && parser->current_item->type == SCCONF_ITEM_TYPE_VALUE) {
-				/* value */
-				parser->state |= STATE_SET;
-				scconf_list_add(&parser->current_item->value.list,
-						      stoken);
+		if ((parser->state & (STATE_VALUE | STATE_SET)) == (STATE_VALUE | STATE_SET)) {
+			scconf_parse_warning_expect(parser, ";");
+			scconf_parse_reset_state(parser);
+		}
+		if (token && *token == '"') {
+			/* quoted string, remove them */
+			token++;
+			len = strlen(token);
+			if (len < 1 || token[len - 1] != '"') {
+				scconf_parse_warning_expect(parser, "\"");
 			} else {
-				/* error */
-				scconf_parse_error_not_expect(parser, stoken);
-			}
-			if (stoken) {
-				free(stoken);
+				/* stoken */
+				stoken = strdup(token);
+				if (stoken) {
+					stoken[len - 1] = '\0';
+				}
 			}
-			stoken = NULL;
 		}
-		break;
+		if (!stoken) {
+			stoken = token ? strdup(token) : NULL;
+		}
+		if (parser->state == 0) {
+			/* key */
+			parser->key = stoken ? strdup(stoken) : NULL;
+			parser->state = STATE_NAME;
+		} else if (parser->state == STATE_NAME) {
+			/* name */
+			parser->state |= STATE_SET;
+			scconf_list_add(&parser->name, stoken);
+		} else if (parser->state == STATE_VALUE &&
+		           parser->current_item->type == SCCONF_ITEM_TYPE_VALUE) {
+			/* value */
+			parser->state |= STATE_SET;
+			scconf_list_add(&parser->current_item->value.list, stoken);
+		} else {
+			/* error */
+			scconf_parse_error_not_expect(parser, stoken);
+		}
+		if (stoken) {
+			free(stoken);
+		}
+		stoken = NULL;
+	} break;
```

Comment:
shouldn't the `break` go on an extra line?

Reply:
probably yes.Or inside of the braces.

Reply:
I rather rewrote this case not to use the block as this did not look like changing anything for me. But added anyway.

Reply:
> please try setting (IndentCaseBlocks)[https://clang.llvm.org/docs/ClangFormatStyleOptions.html#indentcaseblocks] to `true`

Looked again at this option, and I think most of the existing code is using the `false` value. Rewriting everything to match `true` would again lead to a lot of useless indentation. What do you think?